src/components/MAPI/organizations/AccessControl/utils.ts

```diff
@@ -1,5 +1,30 @@
 import * as ui from 'UI/Display/MAPI/AccessControl/types';
 
+const subjectDelimiterRegexp = /\s*(?:[,\s;])\s*/;
+
+/**
+ * Parse subjects from a serialized value (e.g. an user input).
+ * @param from
+ */
+export function parseSubjects(from: string): string[] {
+  const trimmed = from.trim();
+  if (trimmed.length < 1) return [];
+
+  return trimmed.split(subjectDelimiterRegexp);
+}
```

Comment:
This allows us to add multiple subjects at once, dividing them by either `,`,`;`, or any sort of whitespace.